Sources/MongoKitten/ConnectionSettings.swift

```diff
@@ -15,6 +15,18 @@ fileprivate extension Bool {
 
 /// Describes the settings for a MongoDB connection, most of which can be represented in a connection string
 public struct ConnectionSettings: Equatable {
+    /// SSL
+    public enum SSL: Equatable {
```

Comment:
By changing a boolean to an `SSL` instance, a few APIs are likely broken. I don't think it's a huge concern, but you can make the PR a lot simpler by implementing `ExpressibleByBooleanLiteral` here.